internal/api/resend.go

```diff
@@ -129,15 +129,15 @@ func (a *API) Resend(w http.ResponseWriter, r *http.Request) error {
 			if terr != nil {
 				return terr
 			}
-			return a.sendPhoneConfirmation(ctx, tx, user, params.Phone, phoneConfirmationOtp, smsProvider)
+			return a.sendPhoneConfirmation(ctx, tx, user, params.Phone, phoneConfirmationOtp, smsProvider, sms_provider.SMSProvider)
```

Comment:
Hm I wasn't planning on exposing the option to choose channel param to the user on this endpoint as it's not used in the client lib - see description

Reply:
ah i see, yeah we haven't released this to the client lib yet because the projects on gotrue haven't been updated to the version that contains this change